### DISCUSSION TITLE
封禁数大于连接的peer数正常吗？

### DISCUSSION THREAD

**Discussion by CreeperAWA** [PBH-BTN/PeerBanHelper#543] [问答] [answered]
我的统计图表中的趋势显示的封禁数大于连接的peer数，如下图
![PixPin_2024-09-28_12-10-48](https://github.com/user-attachments/assets/6661a305-31c2-470a-b3b5-1be3967a2433)

**Comment by CreeperAWA** [comment]
@Ghost-chu

**Reply by Ghost-chu** [reply, marked as answer]
我不认为这是一个 BUG。

**Reply by CreeperAWA** [reply]
> 我不认为这是一个 BUG。

我认为需要连接才能被检测到，然后才封禁，所以也应当在连接的peer数里
